src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -189,13 +190,14 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
   const chainInfo = getChainInfo(token?.chainId)
   const networkLabel = chainInfo?.label
   const networkBadgebackgroundColor = chainInfo?.backgroundColor
+  const tokenDetailData: any = useTokenDetailQuery(address, 'ETHEREUM')
```

Comment:
Try to get useTokenDetailQuery to return a better type for us to use, rather than typing as any here. you can reference the type derived in TokenPriceQuery